static/stable/itless/modules/fed-modules.json

```diff
@@ -9,7 +9,10 @@
                 "routes": [
                     {
                         "exact": true,
-                        "pathname": "/"
+                        "pathname": "/",
+                        "props": {
+                          "layoutType": "landingPageItless"
```

Comment:
This is the right approach, we just have to make sure the landing page accepts this prop.

We should also add the same prop to the stage stable and prod stable modules configuration and give it the landing page layout as input before we start using it in the UI module itself.

Reply:
@Hyperkid123 landing page had the change merged into master today so it should be good to go. 
Should I just add this prop to the other environments in this PR or would you prefer a separate one?

Reply:
It can be in this one. It will not break anything as it is not really used.

Reply:
@Hyperkid123 all set! Let me know if I need to change anything else.